src/test/Fake.Core.UnitTests/Fake.IO.Globbing.fs

```diff
@@ -60,3 +62,47 @@ let tests =
         finally
           Directory.Delete(testDir, true)
   ]
+
+[<FTests>]
+let toolsTests = 
+  let currentDir =
+    Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)
+  let createUniqueDirectory () =
+      let folder = currentDir </> ((Guid.NewGuid ()).ToString())
+      Directory.CreateDirectory folder |> ignore
+      folder
+  
+  testSequencedGroup "Find tool paths" <|
```

Comment:
Any reason why that needs to be sequenced (not that I think it would matter)?

Reply:
> Because they would conflict when run in parallel due to this search pattern in the impl. `!! ("./**/" @@ toolname)`. Maybe there's a better way to do the tests?

Ah, because you find other results... Yes the "ultimate" solution would be to somehow mock "Environment.CurrentDirectory" but it's fine as it is. Can you add that fact as comment for future reader? Something like `// sequenced because otherwise folders conflict with '!! "./**"'`